src/test/java/uk/gov/GeneratorTest.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov;
+
+import static org.junit.Assert.*;
+import org.junit.*;
+
+public class GeneratorTest {
+	private final String countryJSON = ""
+		+ "{"
+			+ "\"GB\": {"
+				+ "\"index-entry-number\": \"6\","
+				+ "\"entry-number\": \"6\","
+				+ "\"entry-timestamp\": \"2016-04-05T13:23:05Z\","
+				+ "\"key\": \"GB\","
+				+ "\"item\": ["
+					+ "{"
+						+ "\"country\": \"GB\","
+						+ "\"official-name\": \"The United Kingdom of Great Britain and Northern Ireland\","
+						+ "\"name\": \"United Kingdom\","
+						+ "\"citizen-names\": \"Briton;British citizen\""
+					+ "}"
+				+ "]"
+			+ "}"
+		+ "}";
+
+	private final String csv = "";
+
+	private final String expectedPickerJSON = ""
+		+ "{"
+			+ "\"country:GB\": {"
+				+ "\"names\": {"
+					+ "\"en-GB\": \"United Kingdom\""
+				+ "},"
+				+ "\"meta\": {"
+					+ "\"canonical\": true,"
+					+ "\"canonical-mask\": 1,"
+					+ "\"stable-name\": true"
+				+ "},"
+				+ "\"edges\": {"
+					+ "\"from\": []"
+				+ "}"
+			+ "}"
+		+ "}";
+
+	@Test
+	public void run() {
+		Generator generator = new Generator();
+		String output = generator.run(this.countryJSON, this.csv);
```

Comment:
Is this a good pattern for the module? Instantiate it, and then pass in arguments to a method to get the results? Or is it better to construct the instance and then have a getter for the result?

My gut is to keep the class as stateless as possible and rely on the consumer to keep track of state.